### ISSUE TITLE
Encrypting password names and folders/implementing pass-tomb

### ISSUE THREAD

**Issue by wkbt** [IJHack/QtPass#540]
Implementing something similar to https://github.com/roddhjav/pass-tomb which encloses the password-store in an encrypted container would be a great addition to qtpass I believe. Any thoughts?

**Comment by andersaamodt** [comment]
Yes or pass-code at https://github.com/alpernebbi/pass-code. It is not currently possible to use QtPass and one of these extensions at the same time, I think. It might be relatively easy to add support for pass-code because all of the commands are the same, just with "code" added.

**Comment by vuongtuha** [comment]
pass-tomb, pass-coffin, pass-grave, pass-crypt 
I can tell many opt, need qt-pass devs pick one and stick with it for a long term
